index.html

```diff
@@ -124,6 +124,22 @@ <h2>Доп. Информация о расположении и контакты
       <a href="https://htmlacademy.ru">HTML Academy</a>
     </p>
   </footer>
+  <form class="#" action="#" method="post">
+    <button type="button">#</button> <!-- Кнопка закрыть-->
+    <p>
+      <label for="first_and_second_name">Ваше имя:</label>
+      <input id="first_and_second_name" type="text" name="first_and_second_name" value="" placeholder="Имя Фамилия" />
+    </p>
+    <p>
+      <label for="customer_email">Ваш e-mail:</label>
+      <input id="customer_email" type="text" name="customer_email" value="" placeholder="email@example.com" />
```

Comment:
```suggestion
      <input id="customer_email" type="email" name="customer_email" value="" placeholder="email@example.com" />
```
Есть много типов инпутов, нужно их использовать: http://htmlbook.ru/html/input/type